util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopConfigurationProperty.java

```diff
@@ -80,6 +98,35 @@ public String getPassword(Configuration config) {
     return logProperty(lookupKey, value);
   }
 
+  public Map<String, String> getHttpRequestHeaders(Configuration config) throws IOException {
```

Comment:
You don't need to implement this method, `Configuration` class has `getPropsWithPrefix()` method that doing exactly this.

Reply:
Updated, wish I knew this earlier.